src/Microsoft.Management.Infrastructure.CimCmdlets/CimSessionOperations.cs

```diff
@@ -951,7 +951,7 @@ internal void AddSessionToCache(CimSession cimSession, XOperationContextBase con
             CimTestCimSessionContext testCimSessionContext = context as CimTestCimSessionContext;
             uint sessionId = this.sessionState.GenerateSessionId();
             string originalSessionName = testCimSessionContext.CimSessionWrapper.Name;
-            string sessionName = originalSessionName ?? string.Format(CultureInfo.CurrentUICulture, @"{0}{1}", CimSessionState.CimSessionClassName, sessionId);
```

Comment:
Really we should replace `string.Format` with [`string.Create`](https://source.dot.net/#System.Private.CoreLib/src/libraries/System.Private.CoreLib/src/System/String.cs,316)

Reply:
Follow up PR?

Reply:
No, current changes haven't value. Idea is to use _new_ API to reduce allocations. There is no `string.Format` with interpalated string handler. I guess I misled you. Please use `string.Create`.
(You can find all new APIs with interpolated handlers starting with https://source.dot.net/#System.Private.CoreLib/src/libraries/System.Private.CoreLib/src/System/Runtime/CompilerServices/InterpolatedStringHandlerAttribute.cs,382762ca0bf5bcac,references)

Reply:
Also please don't change our trace.WriteLine() and other such callsites - this requires changes in the APIs themselves, which is not always easy. See example #18246.